launcher/main.cpp

```diff
@@ -1,11 +1,13 @@
 #include "StdInc.h"
 #include "mainwindow_moc.h"
 #include <QApplication>
+#include "sdldisplayquery.h"
 
 int main(int argc, char *argv[])
 {
+	auto displayList {getDisplays()};
 	QApplication a(argc, argv);
-	MainWindow w;
+	MainWindow w {displayList};
```

Comment:
Yeah, the usual mistake with `auto foo {bar}` resulting in `std::initializer_list<…>`.